draft-mdt-quic-explicit-measurements.md

```diff
@@ -87,7 +86,30 @@ the fraction of Internet traffic delivered using QUIC is increasing every
 year.  It is imperative to measure packet loss experienced by QUIC users
 directly.
 
-## On-Path Loss Observation
+The Alternate-Marking method {{AltMark}} defines a consolidated method to
+perform packet loss, delay, and jitter measurements on live traffic. However,
+as noted in {{EXPLICIT-MEASUREMENTS}}, applying {{AltMark}} to end-to-end
+transport-layer connections is not easy because packet identification and
+marking by network nodes is prevented when QUIC encrypted transport-layer header
+is being used.
+
+This document aims to define Explicit Host-to-Network Flow Measurement Techniques
```

Comment:
This document defines an extension to QUIC to enable Packet loss measurements.   It is not defining a technique.